torch/csrc/distributed/autograd/context/dist_autograd_context.cpp

```diff
@@ -12,6 +12,17 @@ int64_t DistAutogradContext::context_id() const {
   return context_id_;
 }
 
+std::vector<rpc::WorkerId> DistAutogradContext::getKnownWorkerIds() const {
+  return knownWorkerIDs_;
+};
```

Comment:
Newline between functions. You should run clang-format to take care of this for you. We don't have formatting lint enabled yet, but this will happen some time soon. I'm acting as human linter for the time being...